daemon/graphdriver/zfs/zfs.go

```diff
@@ -99,6 +99,12 @@ func Init(base string, opt []string, uidMaps, gidMaps []idtools.IDMap) (graphdri
 		return nil, fmt.Errorf("BUG: zfs get all -t filesystem -rHp '%s' should contain '%s'", options.fsName, options.fsName)
 	}
 
+	if _, err := os.Stat(base); err != nil && os.IsNotExist(err) {
```

Comment:
Is the `os.Stat` needed here? `os.MkdirAll()` already checks if the path exists (and isn't a file); https://golang.org/src/os/path.go?s=488:535#L9